samples/nextjs/sitecore/config/JssNextWeb.config

```diff
@@ -91,5 +91,20 @@
         </styleguide-image-sample-adaptive>
       </allowedMediaParams>
     </javaScriptServices>
+    <!--
+      Media URLs resolving
+      Tells Sitecore to not include the Sitecore server URL as part of media requests.
```

Comment:
Might be good to explain the why here. e.g. "Tells Sitecore to not include the Sitecore server URL as part of the media requests, so that they are instead routed through Next.js rewrites (see next.config.js). This eliminates exposing the Sitecore server publicly."

Reply:
Changed